### PR TITLE
mgr/dashboard: add some test for controllers/pool.py

### PR DESCRIPTION
**Pull request by wangbo-yw**
add some test for controllers/pool.py

Signed-off-by: wangbo-yw <wangbo2_yewu@cmss.chinamobile.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests
- Teuthology
  - [ ] Completed teuthology run
  - [ ] No teuthology test necessary (e.g., documentation)

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by wangbo-yw** [comment]
@avanthakkar @pereman2 Excuse me, this is my first code submission. The code has been submitted for 8 days but has not been commented, is there a problem with my submission?

**Comment by pereman2** [comment]
> @avanthakkar @pereman2 Excuse me, this is my first code submission. The code has been submitted for 8 days but has not been commented, is there a problem with my submission?

Hey, I'm afraid we missed this PR, my apologies. I'll enable actions and try to review it soon. 

If you may, please take a look at the checklist.

File: src/pybind/mgr/dashboard/tests/test_pool.py [hunk 1]
```diff
@@ -119,3 +120,61 @@ def test_wait_for_pgs_with_waiting(self, _get):
...
+                'rules': [{
+                    'rule_id': 1,
+                    'rule_name': 'test-rule'
+                }]
+            }
+        }[key]
+        Pool._pool_list()
+        mgr.get.assert_called_with('osd_map_crush')
+        self.assertEqual(get_pool_list.call_count, 1)
+        # wit stats
```
Comment (avanthakkar) [inline src/pybind/mgr/dashboard/tests/test_pool.py:146]:
```suggestion
        # with stats
```

**Review by avanthakkar — APPROVED** [review]
LGTM! Just left a comment for a spelling.

**Review by pereman2 — APPROVED** [review]

**Pull request closed without merging**